app/pages/organisation/template.test.js

```diff
@@ -0,0 +1,169 @@
+import { createTestHarness } from '../../test-utils/testHarness';
+import manifest from './manifest.json';
+
+const setup = {
+  template: {
+    path: 'pages/organisation/template.njk',
+  },
+};
+const mockData = {
+  orgId: 'org1',
+  orgName: 'Greater Manchester CCG',
+  odsCode: 'X01',
+  primaryRoleId: 'ID123',
+  address: ['12 Station Road', 'Leeds', 'West Yorkshire', 'LS15 5FG'],
+  isCatalogueAgreementSigned: true,
+  agreementSigned: true,
+  data: [
```

Comment:
Can we actually be more explicit about what this data is. So this will the list of `users` that we supply to the template